prepare.py

```diff
@@ -239,6 +319,19 @@ def set_bin(hist, index, val, err):
             canvas.SetRightMargin(0.24)
             hists[h].Draw('colz text')
             plotPath = os.path.join(plotDir, h)
+            canvas.SetLogy()
+            # START -- format for easier comparison
+            if h == extEffName:
```

Comment:
I think these customizations are going to run into trouble. Instead you can look for what the minimum bin is (in particular I am worried about the "efficiency" histograms, some efficiency bins definitely go below 0.85. example: the barrel gaps in the eta plots). If you have the minimum bin, you can then have 2 options, if min < 0.85, go from 0-1.0, else 0.85-1.0, or similar. Can you please attach a plot of the result of this. Maybe it is fine to have colz truncated at 0.85 (so long as the text appears correctly).